### ISSUE TITLE
Angular2 RC1 - Uncaught No provider for PlatformRef!

### ISSUE THREAD

**Issue by vacarsu** [Urigo/angular2-meteor-auto-bootstrap#10]
Bootstrap causing this error - 

```
Uncaught No provider for PlatformRef!
ReflectiveInjector_._throwOrNull    @   reflective_injector.js:776
ReflectiveInjector_._getByKeyDefault    @   reflective_injector.js:804
ReflectiveInjector_._getByKey   @   reflective_injector.js:767
ReflectiveInjector_.get @   reflective_injector.js:576
createPlatform  @   application_ref.js:41
MeteorApp.bootstrap @   tooltip-options.class.js:22
bootstrap   @   bootstrap.js:9
meteorInstall.node_modules.meteor.conflare-base.client.app.js   @   app.ts:47
fileEvaluate    @   install.js:141
require @   install.js:75
(anonymous function)    @   schema.ts:1
(anonymous function)    @   schema.ts:1
```

**Comment by vacarsu** [comment]
in bootstrap.ts Typescript is saying `Module 'angular2-meteor' has no exported member MeteorApp,
METEOR_PROVIDERS` could this be the cause?

Edit: looks like everything is being imported properly. Not sure what is going on here.

**Comment by Gdust** [comment]
+

**Comment by langfors** [comment]
Try replacing:

`import {bootstrap} from 'angular2-meteor-auto-bootstrap';`

with: 

`import { bootstrap } from '@angular/platform-browser-dynamic';`

in your app.ts.  This is what solved it for me.

**Comment by codebarista** [comment]
For me, the `Error: No provider for PlatformRef!` seems to have been fixed by the recommended TypeScript update.

```
$ meteor update

This project is already at Meteor 1.3.3.1, the latest release.
Your top-level dependencies are at their latest compatible versions.

Newer versions of the following indirect dependencies are available:
 * barbatus:typescript 0.3.1 (0.3.3 is available)
 * barbatus:typescript-compiler 0.5.12 (0.6.1 is available)
To update one or more of these packages, pass their names to `meteor update`.
```

```
$ meteor update barbatus:typescript

Changes to your project's package version selections from updating package
versions:

barbatus:typescript           upgraded from 0.3.1 to 0.3.3
barbatus:typescript-compiler  upgraded from 0.5.12 to 0.6.1
```

**Comment by AlanForester** [comment]
Not working, another solution is exist?

**Comment by AlanForester** [comment]
Problem solved upgrade MeteorJS to 1.3.4 and inserting next packages in the packages.json

> "dependencies": {
>     "@angular/common": "2.0.0-rc.1",
>     "@angular/compiler": "2.0.0-rc.1",
>     "@angular/core": "2.0.0-rc.1",
>     "@angular/platform-browser": "2.0.0-rc.1",
>     "@angular/platform-browser-dynamic": "2.0.0-rc.1",
>     "@angular/router-deprecated": "2.0.0-rc.1",
>     "angular2-meteor": "0.5.5",
>     "angular2-meteor-auto-bootstrap": "0.5.5",
>     "angular2-meteor-polyfills": "^0.1.1",
>     "es6-shim": "^0.35.1",
>     "meteor-node-stubs": "~0.2.0",
>     "reflect-metadata": "0.1.2",
>     "rxjs": "5.0.0-beta.6",
>     "zone.js": "^0.6.12"
>   }

 And changing "as" in routeConfig object to "name"

**Comment by dotansimha** [comment]
Glad to hear that you solved the issue, @AlexCollin, and thanks for sharing your solution!
